conjurapi/config.go

```diff
@@ -24,6 +25,7 @@ type Config struct {
 	AuthnType         string `yaml:"authn_type,omitempty"`
 	ServiceID         string `yaml:"service_id,omitempty"`
 	CredentialStorage string `yaml:"credential_storage,omitempty"`
+	HttpTimeout       int    `yaml:"http_timeout,omitempty"`
```

Comment:
Hi @szh,
I know this merge request is closed for a couple of months, but I need to disagree to limiting timeout configuration to code:
Currently I'm facing timeout issues in summon-conjur. So I'm really happy finding this PR. 
Not allowing to set the client timeout via environment variable requires summon-conjur to be modified. So why not providing configuration options like environment and yaml to this library?
Would you choose to add a timeout configuration to summon-conjur instead of utilizing the advantages offered by the library?